rollup.config.js

```diff
@@ -42,7 +42,7 @@ export default [
     ],
   },
   {
-    input: 'dist/esm/types/index.d.ts',
+    input: 'dist/esm/index.d.ts',
```

Comment:
Is there a reason to change this?

Reply:
After upgrading the packages, no types folder exists in the dist directory. declaration file exists in `dist/esm/index.d.ts`.